docs/articles/systemd.md

```diff
@@ -98,7 +98,7 @@ directory:
 
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H fd:// --graph /mnt/docker-data --storage-driver btrfs
+    ExecStart=/usr/bin/docker daemon -H fd:// --graph /mnt/docker-data --storage-driver=overlay
```

Comment:
I've not been messing around with `--graph`, but looking at the docs - yes it would - I'll fix that too.

Rats... squashing commits always feels yak shavy to me.